CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ have an authority field matching that of the user
 
 - API: Fix a typo ('attachements' -> 'attachments') to ensure compliance with
     the LRS specification and prevent potential silent bugs.
+- Makefile: Fix bug in creating `.ralph/auth.json`
```

Comment:
There is no need to add a CHANGELOG entry for this as it's relevant from a developer-perspective, not users.